### DISCUSSION TITLE
2 Bootloop builds need to remove from sourceforge

### DISCUSSION THREAD

**Discussion by Koushikdey2003** [ROM-builders/temporary#22991] [Q&A]
https://sourceforge.net/projects/rom-builders/files/RMX2020/OctaviOS-v4.6-RMX2020-20230725-0223-VANILLA-Unofficial.zip/download

https://sourceforge.net/projects/rom-builders/files/RMX2020/OctaviOS-v4.6-RMX2020-20230727-1938-GAPPS-Unofficial.zip/download

Idk why got 2 build I only build gapps not vanilla

**Comment by Apon77** [comment]
vanialla came from this one https://cirrus-ci.com/task/4964003504455680

**Reply by Koushikdey2003** [reply]
Hmm yes yes
